server/src/server.ts

```diff
@@ -20,12 +21,13 @@ process.on("SIGINT", () => {
  * Start Express server.
  */
 const server = app.listen(app.get("port"), () => {
-  console.log(
-    "  App is running at http://localhost:%d in %s mode",
+  logger.log(
+    "info",
+    "App is running at http://localhost:%s in %s mode",
     app.get("port"),
     app.get("env")
```

Comment:
Not a blocker, but I think we’d be better off to modernize this line by using a template string instead of interpolation, which is very sensitive to argument order, slower, and requires adding the `splat()` add-on to the logger.

```suggestion
  logger.info(`App is running at http://localhost:${app.get("port")} in ${app.get("env")} mode`);
```